### PR TITLE
astyle, atoum: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/astyle.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+> Indente, formate, et embelli du code source pour des languages de programmation comme C, C++, C# et Java.
+> Pendant l'exécution, une copie du fichier original est créé avec un ".orig" suffixé au nom de fichier original.
+> Plus d'informations : <http://astyle.sourceforge.net/>.
+
+- Applique le style par défaut de 4 espaces pour l'indentation et pas de changement de formatage :
+
+`astyle {{fichier_source}}`
+
+- Applique le style Java avec le style `attached`:
```
Comment (marchersimon) [inline pages.fr/common/astyle.md:11]:
```suggestion
- Applique le style Java avec le style `attached` :
```

File: pages.fr/common/atoum.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# atoum
+
+> Un framework de test unitaire pour PHP simple, moderne et intuitif.
+> Plus d'informations : <http://atoum.org>.
+
+- Initialise un fichier de configuration :
+
+`atoum --init`
+
+- Lance les tests:
```
Comment (marchersimon) [inline pages.fr/common/atoum.md:10]:
```suggestion
- Lance les tests :
```

Comment (Nico385412) [reply pages.fr/common/atoum.md]: i think i should add a rule on the the linter of tldr for this case 🤔 

wdyt ?

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**